src/modules/onlstm/ON_LSTM.py

```diff
@@ -0,0 +1,245 @@
+import torch.nn.functional as F
```

Comment:
If any of this is copied from the ON-LSTM, give credit here.

Reply:
(And in the readme.)

Reply:
You can probably move locked_dropout to utils and get rid of this folder.

Reply:
Style point: We're using all lowercase in filenames.

Reply:
All of these issues are still issues.

Reply:
!

Reply:
I lowercased the F and locked_dropout in separate file in utils. LMK if any other changes required here?